components/Feature/ResidentDetailsForm/index.js

```diff
@@ -77,11 +83,18 @@ const ResidentDetailsForm = ({
       setReferralCompletion({ ...referralCompletion, [serviceId]: result });
       const newReferralSummary = referralSummary.concat([
         {
-          serviceName,
-          serviceId
+          name: serviceName,
+          telephone: serviceTelephone,
+          contactEmail: serviceEmail,
+          referralEmail,
+          address: serviceAddress,
+          websites: serviceWebsites,
+          id: serviceId,
+          referralId: result.id
         }
       ]);
       setReferralSummary(newReferralSummary);
+      setEmailBody(updateEmailBody(undefined, newReferralSummary));
```

Comment:
We're using env vars within the email body, so updateEmailBody more like constructs the string that then gets used to setEmailBody. I think it should be named differently

Reply:
I see... so that function is more like `rebuildEmailBody`? 🤔 I didn't notice the use of environment variables... I assume this comes from somewhere else following the component nesting chain?